packages/endpoints-organizations-registrar/src/entities/organizations/adapters/init-organization-registrar-emails.js

```diff
@@ -255,20 +259,28 @@ Email: ${organization.profile.signatoryEmail}
     },</p>
 <br />
 <br />
-<p>In order to upgrade the functionality of the services you receive, ${
-      caoOrganization.profile.name
-    } has registered ${
-      organization.profile.name
-    } as an entity on the Velocity Network. To learn more about Velocity Network please visit <a href="https://www.velocitynetwork.foundation/" target="_blank">here</a>.</p>
+<p>As part of ongoing service enhancements for ${organization.profile.name}, ${
+      caoOrganization ? caoOrganization.profile.name : ''
```

Comment:
This should be much simpler

caoOrganization?.profile?.name ??
adminName(organization)